docker-compose.yml

```diff
@@ -319,6 +323,16 @@ services:
       - "/config/admin_user_cred.json"
     << : *log-config
 
+  state_groups_cleaner:
+    build: build/state_groups_cleaner
```

Comment:
Ideally this should also be build on CI and use an image (like the other containers are).

Reply:
Makes sense!